src/ScottPlot5/ScottPlot/Colormaps/ArgbColormapBase.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ScottPlot.Colormaps
+{
+    public abstract class ArgbColormapBase : IColormap
+    {
+        public abstract string Name { get; }
+        public abstract uint[] Argbs { get; }
+        public Color GetColor(double intensity, Range? domain)
+        {
+            if (double.IsNaN(intensity))
+            {
+                return Colors.Transparent;
+            }
+
+            domain ??= Range.UnitRange;
+
+            double normalized = domain.Value.NormalizeAndClampToUnitRange(intensity);
```

Comment:
Do you find it confusing that "domain" is of type "range"? This is the "domain range"?

Reply:
Maybe a solution is to name the argument `intensityRange` or something

I'm not finding `domain` particularly intuitive as to what it's supposed to hold

Reply:
Also, such a small nit it's not worth changing, but I'd probably call `normalized` something more descriptive like `colormapFraction`